meta-balena-common/recipes-containers/docker-disk/docker-disk.bb

```diff
@@ -78,10 +79,16 @@ do_compile () {
 
 FILES_${PN} = "/usr/lib/balena/balena-healthcheck-image.tar"
 do_install () {
+	install -d ${D}${sysconfdir}
 	mkdir -p ${D}/usr/lib/balena
 	install -m 644 ${B}/balena-healthcheck-image.tar ${D}/usr/lib/balena/balena-healthcheck-image.tar
+	for image in "${HOSTEXT_IMAGES}"; do
+		echo "${image}" >> ${D}${sysconfdir}/hostapp-extensions.conf
+	done
 }
 
+FILES_${PN} += "/etc/hostapp-extensions.conf"
```

Comment:
Is this like a R/O build artifact or should we bind-mount the file?

Reply:
The idea is that is a RO artifact. This is where the default repository name of the hostapp extensions that have been build-in with the image is stored.
The "update-hostapp-extension" will use this as a last resort so that the system can always be restored to its pristine tested state independently of the hostapp extensions updates.